### ISSUE TITLE
Request: New Release

### ISSUE THREAD

**Issue by jolexa** [gwatts/dyndump#2]
Hello,

I've built the new commits locally since 1.0.0 release and they work well. Would you mind creating a new release?

Thanks!

**Comment by gwatts** [comment]
Yep; i have a few updates coming; will check those in shortly and generate a new release from it

Glad to hear it's working for you!

**Comment by andresvia** [comment]
This is also working for me, and a new release will be well received. Thanks.
